components/network/minio/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2025 Jiwon Na
+#
+
+BUILD_STYLE= justmake
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= minio
+COMPONENT_VERSION= 25.4
+HUMAN_VERSION= 2025-04-22T22-12-26Z
+COMPONENT_GIT_REV=0d7408fc9969caf07de6a8c3a84f9fbb10a6739e
+COMPONENT_SUMMARY= MinIO is a high-performance, S3 compatible object store, open sourced under GNU AGPLv3 license.
+COMPONENT_PROJECT_URL= https://github.com/minio/minio
+COMPONENT_SRC= $(COMPONENT_NAME)-RELEASE.$(HUMAN_VERSION)
+COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL= $(COMPONENT_PROJECT_URL)/archive/refs/tags/RELEASE.$(HUMAN_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:1a63209927b9d1fb239b2a776b7b01d1902eff19379a108f4bc7005edcc0968c
+COMPONENT_FMRI= network/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION= Web Services/Application and Web Servers
+
+include $(WS_MAKE_RULES)/common.mk
+
+CLONEY_MODE=copy
+
+PATH=/usr/gnu/bin:/usr/bin
+
```

Comment:
The need for preferring GNU binaries is common so we have a variable for this:
PATH= $(PATH.gnu)